networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -68,9 +135,11 @@ The release of Network Observability Operator 1.3 deprecates the `spec.Loki.auth
 * Previously, the SCTP, ICMPv4 and ICMPv6 protocols were not covered by the Network Observability agents, resulting in a less comprehensive network flows coverage. These protocols are now recognized to improve the flows coverage. (link:https://issues.redhat.com/browse/NETOBSERV-934[*NETOBSERV-934*])
 
 [id="network-observability-operator-1.3.0-known-issues"]
-=== Known issue
+=== Known issues
 * When `processor.metrics.tls` is set to `PROVIDED` in the `FlowCollector`, the `flowlogs-pipeline` `servicemonitor` is not adapted to the TLS scheme. (link:https://issues.redhat.com/browse/NETOBSERV-1087[NETOBSERV-1087])
```

Comment:
Rather than removing it from here, I think that we would need to document this as a bug fix for 1.4 then. Because it was indeed a Known Issue at the 1.3 release, I don't think we can remove it. But we definitely should document that it is now fixed since it was a published known issue. 

@OlivierCazade can you please add a Release Note text for this bug: https://issues.redhat.com/browse/NETOBSERV-1087?

Reply:
I just need to know the basic "What happens now?". The Bug fix will be like this:
Previously, when `processor.metrics.tls` was set to `PROVIDED` in the `FlowCollector`, the `flowlogs-pipeline` `servicemonitor` did not adapt to the TLS scheme. Now, <what I need to know>.